### PR TITLE
docker compose with all services

### PR DESCRIPTION
**Pull request by voutasaurus**
Signed-off-by: Anthony Voutas <anthonyv@kiva.org>

### CODE REVIEW

**Comment by voutasaurus** [comment]
todo: put the env variables into the dockerfile

File: docker-compose.all.yml [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+      - AUTH0_ALGORITHM=RS256
+      - AUTH0_DOMAIN=kiva-protocol-standalone.auth0.com
+      - POSTGRES_HOST=protocol-identity-db
+      - POSTGRES_DB=identitydb
+      - POSTGRES_PORT=5432
+      - POSTGRES_USER=dbuser
+      - POSTGRES_PASSWORD=dbpass
+
+  demo-controller:
+    image: "kivaprotocol/demo-controller:0.0.42"
```
Comment (jsaur) [inline docker-compose.all.yml]: We're moving away from demo-controller image and on to aries-controller image

**Review by jsaur — COMMENTED** [review]
One comment about demo-controller

**Review by jsaur — APPROVED** [review]
Looks good 👍

**Pull request merged**